sdk/include/reactos/undocuser.h

```diff
@@ -204,6 +204,20 @@ BOOL WINAPI PaintMenuBar(HWND hWnd, HDC hDC, ULONG left, ULONG right, ULONG top,
 #define DrawCaptionTemp DrawCaptionTempA
 #endif
 
+//
+// Hard error balloon package
+//
+typedef struct _BALLOON_HARD_ERROR_DATA
+{
+    DWORD cbHeaderSize;
+    DWORD Status;
+    DWORD dwResponseType;
```

Comment:
Do you actually know what this corresponds to?

Reply:
The name is wrong, but it is the third parameter passed to NtRaiseHardError as shown here:
https://github.com/learn-more/test-utils/blob/master/HardError/HardError.cpp#L106

Its value can be any combination of the enumerations HardErrorResponseButton , HardErrorResponseDefaultButton , HardErrorResponseIcon , HardErrorResponseOptions  shown here: https://github.com/learn-more/test-utils/blob/master/HardError/HardError.cpp#L32

Reply:
So dwResponseType sounds OK. Or use "dwValidResponseOptions" or some variation on it ^^